edge/discussion_materials/ONNX Edge scope and profile definition.md

```diff
@@ -0,0 +1,72 @@
+# ONNX Edge scope and profile definition
+Author: Yedong Liu, Milan Oljaca, Saurabh Tangri
+
+## Scope of Edge
+
+Edge computing is computing that’s done at or near the source of the data, instead of relying on the cloud at one of a dozen data centers to do all the work. Edge computation is largely or completely performed on distributed edge devices. The target of edge computing is any application or general functionality needing to be closer to the source of the action where system can process and interact with the real world instantly.
```

Comment:
Since I cannot simply highlight some text to comment on, I will propose changes to the paragraph as a whole. Also, I am thinking this is where some diagrams would be good.

Consider rewriting as follows:
Broadly speaking, scope of edge can be defined as "not cloud". Edge computing is computing that’s done at or near the source of the data, without relying on the cloud computing resources in data centers. Edge computation is largely or completely performed on distributed edge devices. The consumer of edge computing is any application or general functionality needing to be closer to the source of the action where system can process and interact with the real world instantly.